docs/sentry/nestjs.md

```diff
@@ -0,0 +1,266 @@
+# Sentry Nestjs
+
+
+### Getting Started
+Install all needed dependencies:
+* [@sentry/node](https://www.npmjs.com/package/@sentry/node)
+* [@sentry/integrations](https://www.npmjs.com/package/@sentry/integrations)
+
+```bash
+npm i @sentry/node @sentry/integrations
+#or
+yarn add @sentry/node @sentry/integrations
+```
+
+Add the following to your tsconfig
+```json
+"sourceMap": true,
+"inlineSources": true,
+"sourceRoot": "/",
+```
+
+### Setting up the SDK
+You need to create some Filters and Modules to have automatic capturing enabled. Create the following files:
+
+
+---
```

Comment:
Is it a good idea to have these files as gist somewhere that can be easily downloaded / linked via a command? Not sure yet, since these are subject to change or might not fit in the code structure of the corresponding project. Let me know what you think

Reply:
Yes I think this would be cool.
Do you know how to make gists for an organization? I only find the option to create one under my private account

Reply:
No but we can create a gist for the nanogiantsgmbh account. Create the gist in your account and I move it to the nanogiants main accoutn